src/main/java/mekanism/common/Mekanism.java

```diff
@@ -1516,11 +1516,7 @@ public void init(FMLInitializationEvent event)
 				}
 			}
 		}
-		if(Loader.isModLoaded("EnderIO") || Loader.isModLoaded("GalacticraftCore"))
-		{
-			isSiliconLoaded = true;
-		}
-		if(Loader.isModLoaded("ProjRed|Core"))
+		if(Loader.isModLoaded("EnderIO") || Loader.isModLoaded("GalacticraftCore") || Loader.isModLoaded("ProjRed|Core"))
```

Comment:
This should have a config if we are supporting the mod for community use.

It can be easily done by adding:
`if (MekanismConfig.general.enableSiliconCircuitRecipe)`
Above
`if(Loader.isModLoaded("EnderIO") || Loader.isModLoaded("GalacticraftCore") || Loader.isModLoaded("ProjRed|Core"))`
and implement it as a config.

Reply:
already made one lel... not pushed yet